react-course/video-26/demo.html

```diff
@@ -0,0 +1,35 @@
+<body>
+    <div id="root"></div>
+    <script src="https://unpkg.com/react@16.12.0/umd/react.development.js"></script>
+    <script src="https://unpkg.com/react-dom@16.12.0/umd/react-dom.development.js"></script>
+    <script src="https://unpkg.com/@babel/standalone@7.8.3/babel.js"></script>
+    <script src="https://unpkg.com/react-error-boundary@1.2.5/dist/umd/react-error-boundary.js"></script>
+    <script type="text/babel">
+        function PokemonInfo({ pokemonName }) {
+            return <div>Need to request info for: {pokemonName} </div>
+        }
+
+        function App() {
+            const [pokemonName, setPokemonName] = React.useState('')
+
+            function handleSubmit(event) {
+                event.preventDefault()
+                setPokemonName(event.target.elements.pokemonName.value)
```

Comment:
Is the `event.target` structure clear? Should we go over it in the next session?

Reply:
I think that basically, it retrieves the value of whatever input it was called on. In this case, it’s my input element so whatever I insert in my input can be accessed through event.target.value, here the pokemon name will be whatever the user writes in the input.